pandas/core/strings.py

```diff
@@ -898,23 +898,92 @@ def str_join(arr, sep):
 
 def str_findall(arr, pat, flags=0):
     """
-    Find all occurrences of pattern or regular expression in the
-    Series/Index. Equivalent to :func:`re.findall`.
+    Find all occurrences of pattern or regular expression in the Series/Index.
+
+    Equivalent to applying :func:`re.findall` to all the elements in the
+    Series/Index.
 
     Parameters
     ----------
     pat : string
-        Pattern or regular expression
-    flags : int, default 0 (no flags)
-        re module flags, e.g. re.IGNORECASE
+        Pattern or regular expression.
+    flags : int
+        re module flags, e.g. re.IGNORECASE (default is 0, which means
```

Comment:
can you put `re` in double backtick quotes?

Reply:
Note that the docstring guide shows only single backtick quotes at the moment.

Reply:
Then we should update that. In principle (according to numpydoc format spec): single backtick quotes for refering to keyword arguments (eg if you would refer to `` `flags` `` in the text somewhere), double backticks for actual code snippets (eg if you would say somewhere ``` ``flags=0`` ```, and I would consider stdlib module names also as code)